features/checkout/paying_for_order/preventing_payment_step_completion_without_a_selected_method.feature

```diff
@@ -0,0 +1,53 @@
+@checkout
+Feature: Preventing payment step completion without a selected method
+    In order to be prevent from finish payment step without selected method
```

Comment:
the above + `from finishing the payment step without selecting a method` / `from finishing the payment step with no method selected`